.github/ISSUE_TEMPLATE/2-regression-report.yaml

```diff
@@ -0,0 +1,62 @@
+name: Regression Report
+description: Report a regression in Hermione
+labels: ["type: regression"]
+body:
+    - type: input
+      id: last-working-version
+      attributes:
+          label: Last Hermione version that worked
+          placeholder: e.g. 8.0.3
+      validations:
+          required: true
```

Comment:
We can set this field as optional - if a minimal case is provided, we can find the broken version on our own